### DISCUSSION TITLE
How to remove an existing image sha

### DISCUSSION THREAD

**Discussion by ns-ggeorgiev** [renovatebot/renovate#32891] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### Please tell us more about your question or problem

I have a corpus - docker registry that I have guarantee that every tag is immutable. That makes using sha when referring to images from that corpus pointless. I would like to be able to setup renovate to drop the digest when updating the image, but I cannot find how.

With other words, if I have 

`my-corpus/repo:1.2.3@sha256:9de8526c502ef477c4ca8f60d52cc01f74b9eb2187336e141c50d113c500b8ba`

should be replaced with 

`my-corpus/repo:1.2.4`

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
It might somehow be possible with existing config options, but "unpinning" digests is not an official feature which renovate supports
